enzyme/Enzyme/Utils.h

```diff
@@ -1044,6 +1046,8 @@ template <typename T> static inline llvm::StringRef getFuncNameFromCall(T *op) {
       return called->getFnAttribute("enzyme_math").getValueAsString();
     else if (called->hasFnAttribute("enzyme_allocator"))
       return "enzyme_allocator";
+    else if (called->hasFnAttribute("enzyme_notypeanalysis"))
```

Comment:
This should not be here, this is for use in adjointgenerator/etc. You should modify TypeAnalysis to do this check.